### DISCUSSION TITLE
Backporting 18.19.1 security fixes to 18.18.x

### DISCUSSION THREAD

**Discussion by andrewretool** [nodejs/help#4646] [Q&A] [answered]
Hello, is there a reason the security fixes for CVEs below were applied to 18.19.1 but not 18.18.x? Do they not apply to 18.18.x?

- CVE-2024-21892
- CVE-2024-22019
- CVE-2023-46809
- CVE-2024-22025

**Comment by atlowChemi** [comment, marked as answer]
Hi @andrewretool 
No, they will most likely not be applied. Once 18.19.X has been released, patches are not applied backwards on minors, but only on majors.
